WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Stats/DashboardStatsNudgeView.swift

```diff
@@ -1,8 +1,25 @@
 import UIKit
 
-final class DashboardStatsNudgeButton: MultilineButton {
+final class DashboardStatsNudgeView: UIView {
 
-    var onTap: (() -> Void)?
+    var onTap: (() -> Void)? {
+        didSet {
+            let tapGesture = UITapGestureRecognizer(target: self, action: #selector(buttonTapped))
+            addGestureRecognizer(tapGesture)
+        }
+    }
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = WPStyleGuide.fontForTextStyle(.subheadline)
+        label.textColor = .textSubtle
```

Comment:
I think we can add `isAccessibilityElement = true` and `accessibilityTraits = .button` here. Wdyt?